common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -97,6 +97,15 @@ class ProctoringFields(object):
         scope=Scope.settings,
     )
 
+    hide_after_due = Boolean(
+        display_name=_("Hide Exam Results After Due Date"),
+        help=_(
+            "This setting overrides the default behavior of showing exam results after the due date has passed."
```

Comment:
Thanks, Eric. No need to spend time tracking this down, unless you really want to! :-)